services/self-service/src/main/java/com/epam/datalab/backendapi/dropwizard/listeners/MongoStartupListener.java

```diff
@@ -38,50 +40,66 @@
 import java.util.List;
 import java.util.Set;
 import java.util.TreeSet;
+import java.util.stream.Collectors;
 
+import static java.lang.String.format;
 import static java.util.Comparator.comparing;
 import static java.util.stream.Collectors.collectingAndThen;
 import static java.util.stream.Collectors.toCollection;
 
 @Slf4j
 public class MongoStartupListener implements ServerLifecycleListener {
 
-    private static final String PATH_TO_GENERAL_ROLES = "/mongo/general/mongo_roles.json";
+    private static final String PATH_TO_ROLES = "/mongo/%s/mongo_roles.json";
     private static final ObjectMapper MAPPER = new ObjectMapper();
     private final UserRoleDAO userRoleDao;
     private final SelfServiceApplicationConfiguration configuration;
     private final SettingsDAO settingsDAO;
+    private final EndpointDAO endpointDAO;
 
     @Inject
     public MongoStartupListener(UserRoleDAO userRoleDao, SelfServiceApplicationConfiguration configuration,
                                 SettingsDAO settingsDAO, EndpointDAO endpointDAO) {
         this.userRoleDao = userRoleDao;
         this.configuration = configuration;
         this.settingsDAO = settingsDAO;
+        this.endpointDAO = endpointDAO;
     }
 
     @Override
     public void serverStarted(Server server) {
         settingsDAO.setServiceBaseName(configuration.getServiceBaseName());
         settingsDAO.setConfOsFamily(configuration.getOs());
         settingsDAO.setSsnInstanceSize(configuration.getSsnInstanceSize());
+        List<EndpointDTO> endpointDTOs = endpointDAO.getEndpoints();
         if (userRoleDao.findAll().isEmpty()) {
-            log.debug("Populating DataLab roles into database");
-            userRoleDao.insert(getRoles());
+            log.info("Populating DataLab default roles into database");
+            List<UserRoleDTO> populatingRoles = getRoles(CloudProvider.GENERAL);
```

Comment:
Please rename this variable **populatingRoles** to **generalRoles** for example.